overlord/overlord.go

```diff
@@ -167,6 +168,36 @@ func (o *Overlord) Stop() error {
 	return err1
 }
 
+// Settle runs first a state engine Ensure and then wait for activities to settle.
+// That's done by waiting for all managers activities to settle while
+// making sure no immediate further Ensure is scheduled. Chiefly for tests.
+// Cannot be used in conjuction with Run.
+func (o *Overlord) Settle() {
+	o.ensureLock.Lock()
+	if o.ensureTimer != nil {
+		panic("cannot use Settle with an ensure loop (Overlord.Run)")
```

Comment:
`"cannot use Settle concurrently with other Settle or Loop calls"`

Let's rename Run to Loop please.. Run doesn't make clear if it's once or multiple times (Settle also "runs").